Makefile.core.mk

```diff
@@ -99,13 +99,18 @@ opt:
 clean:
 	@rm -fr resources .htmlproofer tmp generated public out samples install go tests/integration/ manifests
 
-lint: clean_public build_nominify lint-copyright-banner lint-python lint-yaml lint-dockerfiles lint-scripts lint-sass lint-typescript lint-go
+# Use docs-lint-yaml as a target around the common-files lint-yaml, but set +o pipefail for that call
+# otherwise it fails.
+docs-lint-yaml:
+	@set +o pipefail; $(MAKE) -f common/Makefile.common.mk lint-yaml
```

Comment:
This is a little surprising ... how does setting this for the lint target affect tests? The tests aren't dependent on this target, are they?

Reply:
You will noticed that I removed ~line 200 which was setting this for all the targets, and then just added it back here for the single target that needs it from the istio.io repo's perspective.